apps/organisations/admin.py

```diff
@@ -3,7 +3,27 @@
 from . import models
 
 
+class MemberInline(admin.TabularInline):
+    model = models.Member
+    fields = ['member',]
```

Comment:
polylint says:
`apps/organisations/admin.py:8:23: E231 missing whitespace after ','`

Reply:
right! That was the thing I also wanted to investigate. :roll_eyes: